xapian-core/tests/api_cluster.cc

```diff
@@ -51,6 +51,58 @@ make_stemmed_cluster_db(Xapian::WritableDatabase &db, const std::string &)
     }
 }
 
+/** Round Robin clusterer:
+ *  This clusterer is a minimal clusterer which will cluster documents as -
+ *  ith document goes to the (i % k)th cluster where k is the number of clusters and
+ *  0 <= i < N; where N is the number of documents
+ */
+class XAPIAN_VISIBILITY_DEFAULT RoundRobin : public Xapian::Clusterer {
```

Comment:
Oh I see. I guess my confusion arises mainly because bitbucket handles this in a different way. Thanks @ojwb!